spec/models/participant_spec.rb

```diff
@@ -0,0 +1,50 @@
+
+require 'rails_helper'
+
+
+describe Participant do
+
+  before(:each) do
+    @student = create(:student, name: "Student", fullname: "Student Test", email: "student1@gmail.com" , password: "123456", password_confirmation: "123456")
```

Comment:
It is overkill to use `create` in unit tests.It would be better to use `double` or `build`.